### ISSUE TITLE
Add tests for refactored `Color` class

### ISSUE THREAD

**Issue by BrorSebastianSjovald** [DD2480-Group-10/manim#5]
When `Color` has received it's new functions, they should be tested. This can be done by refactoring old tests for the functions or adding new ones.

Blocked by #1

**Comment by amandakrohn** [comment]
This is no longer relevant since we will not have a `Color` class. I will close the issue.
